services/common/src/interfaces/mineIncident.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface IMineIncident {
+  mine_name: string;
+  categories: any;
+  incident_timestamp: any;
```

Comment:
Can you please change the data type here to string ?

Reply:
Updated to string.